blosc/blosc2.c

```diff
@@ -2185,10 +2197,10 @@ int blosc2_getitem_ctx(blosc2_context* context, const void* src, int start,
 }
 
 /* execute single compression/decompression job for a single thread_context */
-static void t_blosc_do_job(void *ctxt)
-{
-  struct thread_context* thcontext = (struct thread_context*)ctxt;
-  blosc2_context* context = thcontext->parent_context;
+static void t_blosc_do_job(void *ctxt) {
+
+  struct thread_context *thcontext = (struct thread_context *) ctxt;
+  blosc2_context *context = thcontext->parent_context;
```

Comment:
In c-blosc2 we follow the convention that `*` are put at the end of the type qualifier, not the beginning of the variable.  I know that in Caterva we follow the latter convention, but please try to adapt to the different conventions till we have time to unify them.